README.md

```diff
@@ -49,6 +49,13 @@ Debian packages [are available](https://packages.debian.org/sid/net/prometheus).
 
 Docker images are available on [Quay.io](https://quay.io/repository/prometheus/prometheus).
 
+You can launch a Prometheus container for trying it out with
+```
```

Comment:
To be consistent with the other codeboxes in this README.md, please use a 4-space indent and instead of 3 backticks, start command lines with `$`, and put a colon after "with:".